recipes/qt/yum_requirements.txt

```diff
@@ -0,0 +1,8 @@
+libX11-devel
+libXt-devel
+libXext-devel
+libXrender-devel
+gtk2-devel
+dbus-devel
+libXi-devel
+mesa-libGLU-devel
```

Comment:
`gtk2-devel` is required to have `gtkstyle` built. But it contains the Gtk2 header files (if I understand things correctly :-).